### ISSUE TITLE
tsParticles - Noticable stutter and lag on Firefox?

### ISSUE THREAD

**Issue by QuiteAFancyEmerald** [tsparticles/tsparticles#3089]
### Contact Details

emerald@titaniumnetwork.org

### What do you need?

Hi! So I've been noticing a stutter that only seems to happen on Firefox. I've tried this out on multiple devices and it just seems to be an issue with `tsParticles`. I should also note that all the demos here also have that lag and stutter.

On Chrome, `tsParticles` is buttery smooth and the `fpsLimit` option works great but it seems to be the opposite on Firefox.

Thank you :D

### Version

1.39.2

### Which library are you using?

Vanilla (tsparticles)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by matteobruni** [comment]
Hi @QuiteAFancyEmerald,

I installed Firefow now but I didn't notice any difference in performance from Chrome or Safari

**Comment by QuiteAFancyEmerald** [comment]
Interesting. Could it perhaps be with Firefox Developer Edition only?

**Comment by accidental-feature** [comment]
I use Firefox developer addition as well. I would look at the performance section in the dev tools to see if there is a problem found there. I personally feel like it runs very well on Firefox compare to some other browsers.

**Comment by QuiteAFancyEmerald** [comment]
Yeah super weird. I can't seem to pin this down since I have tested this on multiple devices now (all Windows however, latest version of Firefox Developer Edition). 

I looked into this more and the issue seems to exist when syncing a Firefox account which has been in use for awhile. When you install a fresh copy no issues seem to exist. Not sure if this is related to extensions, etc. but I'm mostly assuming this is a Firefox issue since yeah.
